website-custom-views/src/content/api-reference/commercetools-frontend-cypress.mdx

```diff
@@ -73,37 +71,24 @@ import '@commercetools-frontend/cypress/add-commands';
 
 # Commands
 
-## cy.loginToMerchantCenter
+## cy.loginToMerchantCenterForCustomView
 
-This command performs the user login to the Merchant Center. It automatically detects whether the Custom View is running on localhost or production and chooses the appropriate login mechanism.<br/>
-
-- When it runs locally, the same mechanism used in the `cy.loginByOidc` is used.
-- When it runs on production, a normal login flow is used where the user enters the credentials into the login form.
+This command performs the user login to the Merchant Center to test a Custom View. Testing Custom Views is expected to occur on `localhost`, therefore the `cy.loginByOidc` mechanism is used.
```

Comment:
I'm wondering whether it makes sense to mention the `cy.loginByOidc` command here since we don't talk about it in the Custom Views docs (it's more like an internal detail to me).